app/src/main/java/com/example/simplenavigation/MainActivity.java

```diff
@@ -25,28 +25,56 @@
     private Robot mRobot;
     private final String TAG = MainActivity.class.getSimpleName();
 
+    /**
+     * Function getInt is used to get integer value from EditText views
+     * @param viewId is used to find view we want to get value from
+     * @return is value of view or if view is empty function returns "50" so robot ends up facing upwards towards user
+     */
+    public int getInt(int viewId) {
+        EditText text = findViewById(viewId);
+        String tiltVal = text.getText().toString();
+        if (tiltVal.equals("")) {
+            text.setText("50");
+            return 50;
```

Comment:
I see why you're returning 50, but this isn't very good practice.

It's confusing to the reader because your function is called `getInt()`, but you're returning 50 when the value is empty. Intuitively, one would expect it to return 0 and therefore tilt the value to 0.

Reply:
Function name changed to **getTilt** to make it more clear why is it returning specific value.